Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.mm

```diff
@@ -383,7 +384,7 @@ static CGFloat effectivePointsPerMeter(CALayer *caLayer)
     updateOpacity();
     startAnimating();
 
-    send(Messages::ModelProcessModelPlayer::DidFinishLoading());
+    send(Messages::ModelProcessModelPlayer::DidFinishLoading(WebCore::FloatPoint3D(m_originalBoundingBoxCenter.x, m_originalBoundingBoxCenter.y, m_originalBoundingBoxCenter.z), WebCore::FloatPoint3D(m_originalBoundingBoxExtents.x, m_originalBoundingBoxExtents.y, m_originalBoundingBoxExtents.z)));
```

Comment:
Do we have a facility to create a FloatPoint3D directly from a simd_float3? Might be a good one to add